examples/wireguard.yml

```diff
@@ -1,6 +1,6 @@
 kernel:
-  image: linuxkit/kernel:4.9.43
-  cmdline: "console=tty0 console=ttyS0 console=ttyAMA0"
+  image: linuxkit/kernel:4.9.49
+  cmdline: "console=ttyS0"
```

Comment:
can we not change the `console=` line - although it can be annoying for debug, this is generally what we use so it works everywhere...

Reply:
Yes, no problem, it doesn't make a difference to me. I'll submit a change now. However, things are not as consistent as you say:

```
linuxkit/examples $ grep -hr 'cmdline:.*console=' | sort | uniq -c | sort -nr
      7   cmdline: "console=ttyS0"
      6   cmdline: "console=tty0 console=ttyS0 console=ttyAMA0"
      2   cmdline: "console=tty0 console=ttyS0"
      1   cmdline: "console=ttyS1"
      1   cmdline: "console=tty0"
```